packages/pluggableWidgets/listview-swipe-native/package.json

```diff
@@ -21,7 +21,7 @@
   "dependencies": {
     "@mendix/piw-native-utils-internal": "*",
     "@mendix/piw-utils-internal": "*",
-    "react-native-gesture-handler": "1.8.0"
+    "react-native-gesture-handler": "1.10.3"
```

Comment:
is there a reason that we are still using V1 ?

Reply:
React-navigation :)